### ISSUE TITLE
Video reproduction freezes on Ozone Wayland when started in VPN split tunnelling

### ISSUE THREAD

**Issue by Digitalone1** [flathub/com.brave.Browser#683]
This is a weird issue, but I'd like to report it anyway.

I use Mullvad VPN and sometimes I start Brave Browser in split tunnelling mode (from Mullvad app). I use the Flatpak version of Brave and the `--ozone-platform=wayland` flag (set in `brave://flags/` since I don't like the brurry UI on fractional scaling).

In the conditions abovementioned, video reproduction stops after an amount of time (40-50 seconds). Steps to reproduce:

1. Start a VPN connection through Mullvad
2. Open Brave Flatpak in split tunnelling mode from Mullvad UI
3. Go [here](http://s3.videorx.com/sm/2017-10-H264-Encoder-Comparison/fast-motion/5000kbps/talonG2-720p-5000Kbps_2017-08-15-17.30.20.572-PDT_0.mp4) and start the video
4. The video stops after 40-50 seconds
5. No other video can be reproduced

This is always reproducible, unless you start Brave Flatpak normally (through the system app menu, not from Mullvad UI). I tried to reproduce this issue on the [AUR](https://aur.archlinux.org/packages/brave-bin) version, but it's working properly there, so it's specific to the Flatpak version.

I thought this was a Mullvad issue relying on how the browser is started (I reported it [here](https://github.com/mullvad/mullvadvpn-app/issues/7045)), but later I found out that if I revert the `ozone-platform` flag to the default (X11 and blurry UI), the issue is not reproducible even when the browser is started in the split tunnel from Mullvad UI.

So it's an issue with the Ozone Wayland platform that happens only in Flatpak version. Maybe this is reproducible also with Chromium Flatpak (but I couldn't test it). Maybe you don't care to fix it since it happens in specific conditions when started from a VPN app you don't use, but I though it was worth to report the issue anyway.

Any thoughts on why this happens? Thanks.

**Comment by Digitalone1** [comment]
> Maybe this is reproducible also with Chromium Flatpak (but I couldn't test it).

I tested Chromium Flatpak under the same conditions. Ozone Wayland and split tunnelling enabled, but the issue does not occur. So it seems something related to Brave.

**Comment by rany2** [comment]
Does this still occur? I don't use Mullvad so I can't test it myself, but I have no issues watching videos in Brave with Ozone enabled while using Wireguard.

**Comment by Digitalone1** [comment]
> I'm guessing that Mullvad split tunneling uses network namespaces or something that interferes with Flatpak sandboxing. At any rate, it probably should be something you report to https://github.com/flatpak/flatpak instead; likely with some technical details as to how Mullvad split tunneling works.

I don't think it would help reporting this issue to flatpak. What flatpak can do? They would suggest me to report it to Brave, as I already did. I don't know if it still occurs since I gave up using the Flatpak version because of this issue more than a year ago. Maybe it's just that sandboxes browsers can't be used with split tunneling...
